sections/futures/Trades/TimeDisplay.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC, useState } from 'react';
+import { format, Locale } from 'date-fns';
+
+import getLocale from './getLocale';
+
+type TimeDisplayProps = {
+	cellPropsValue: any;
+};
+
+const TimeDisplay: FC<TimeDisplayProps> = ({ cellPropsValue }) => {
+	const [show12hr, setShow12h] = useState<boolean>(false);
+
+	const handleOnClick = () => {
+		setShow12h(!show12hr);
+	};
+
+	let language: Locale = { code: '' };
+
+	navigator.languages !== undefined
+		? (language.code = navigator.languages[0])
+		: (language.code = navigator.language);
+
+	const date = format(new Date(cellPropsValue), 'MM/dd/yy', {
+		locale: getLocale(language.code),
+	});
+	const time12hr = new Date(cellPropsValue).toLocaleTimeString(language.code);
+	const time24hr = format(new Date(cellPropsValue), 'HH:mm:ss', {
```

Comment:
so looking into some options in regards to the `format`, im wondering if we can just use [intlFormat](https://date-fns.org/v2.28.0/docs/intlFormat) here and pass the locale to that?